cmd/fleetctl/package.go

```diff
@@ -240,6 +240,12 @@ func packageCommand() *cli.Command {
 				EnvVars:     []string{"FLEETCTL_DISABLE_KEYSTORE"},
 				Destination: &opt.DisableKeystore,
 			},
+			&cli.StringFlag{
+				Name:        "osquery-db",
+				Usage:       "Sets a custom osquery database directory (requires orbit >= v1.22.0)",
```

Comment:
This must be an absolute path, right? User can't do: `./my-osquery.db`, or should we support relative paths?

Reply:
Yeah, should be absolute, as orbit runs as root, on linux the above will basically use `/my-osquery.db`.

Reply:
I added a validation check to make sure the provided path is absolute.

Reply:
[Prevent non-absolute paths](https://github.com/fleetdm/fleet/pull/16554/commits/e52a8db46e0f1d60ae80f2d5ffb4eee054012733)